tests/sentry/sentry_apps/services/test_model.py

```diff
@@ -0,0 +1,42 @@
+from sentry.sentry_apps.services.app.serial import serialize_sentry_app_installation
+from sentry.sentry_apps.services.app.service import app_service
+from sentry.testutils.cases import TestCase
+from sentry.testutils.silo import region_silo_test
+
+
+@region_silo_test
+class TestRpcApiApplication(TestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.user = self.create_user(name="foo")
+        self.org = self.create_organization(owner=self.user)
+        self.project = self.create_project(slug="boop", organization=self.org)
+
+        self.sentry_app = self.create_sentry_app(
+            name="foo", organization=self.org, webhook_url="https://example.com", scopes=()
+        )
+
+        self.install = self.create_sentry_app_installation(
+            slug="foo", organization=self.org, user=self.user
+        )
+
+    def test_serializes_correct_fields_helper(self):
+        install = serialize_sentry_app_installation(self.install, self.install.sentry_app)
+
+        assert (
+            f"this {install.sentry_app.application} is so skibidi"
+            == f"this id={install.sentry_app.application_id} is so skibidi"
```

Comment:
Since we set the `repr` to False on `RpcApiApplication` for the `client_id` & `client_secret` fields the only field left to print is the id.